### DISCUSSION TITLE
Nuxt formkit playground?

### DISCUSSION THREAD

**Discussion by gbyesiltas** [formkit/formkit#974] [General]
Hey, I wanted open an issue for something I'm encountering and I found [this playground link](https://formkit.com/playground?fkv=latest&fileTab=Playground.vue&css-framework=genesis) which is very convenient for creating formkit reproductions in Vue. I was wondering if there is one for Nuxt as well. Thanks! :)

**Comment by justin-schroeder** [comment]
There is not @gbyesiltas — best case would be to use something like stackblitz

**Reply by gbyesiltas** [reply]
Got you, thanks :)
